CHANGELOG.md

```diff
@@ -20,6 +20,10 @@
   - [changelog](https://github.com/getsentry/sentry-cli/blob/master/CHANGELOG.md#220)
   - [diff](https://github.com/getsentry/sentry-cli/compare/1.71.0...2.2.0)
 
+### Fixes
+
+- Correctly get native stack trace on Unity 2020 ([#836](https://github.com/getsentry/sentry-unity/pull/836))
+
```

Comment:
```suggestion
```
Since this has not been released yet it did technically not break. No break - no fix :D